annotations-test/pom.xml

```diff
@@ -9,9 +9,9 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <maven.compiler.source>21</maven.compiler.source>
-        <maven.compiler.target>21</maven.compiler.target>
-        <java.version>21</java.version>
+        <maven.compiler.source>17</maven.compiler.source>
```

Comment:
this change was already reverted. Do not change java version to 17. Leave as it was. Revert for all pom.xml files